tests/Import_scanner_test.py

```diff
@@ -42,13 +45,13 @@ def test_check_test_file(self):
                 missing_tests += [test]
 
         if len(missing_tests) > 0:
-            print('The following scanners are missing test cases or incorrectly named')
-            print('Names must match those listed in /dojo/tools')
-            print('Test cases can be added/modified here:')
-            print('https://github.com/DefectDojo/sample-scan-files\n')
+            logger.info('The following scanners are missing test cases or incorrectly named')
+            logger.info('Names must match those listed in /dojo/tools')
+            logger.info('Test cases can be added/modified here:')
+            logger.info('https://github.com/DefectDojo/sample-scan-files\n')
             for test in missing_tests:
-                print(test)
-            print()
+                logger.info(test)
+            logger.info()
```

Comment:
We can't use `logger.info()` this way and I'm not sure if logging a simple "\n" is necessary here?
```suggestion

```